### ISSUE TITLE
After a change to content, editor loses language.

### ISSUE THREAD

**Issue by elevatebart** [koca/vue-prism-editor#30]
Great tool !! Thank you.

Just a little minor quirk.
On load, the DOM looks like this:

```html
<pre spellcheck="false" autocapitalize="off" autocomplete="off" autocorrect="off" data-gramm="false" class="prism-editor__code  language-html" style="border-top-left-radius: 0px; border-bottom-left-radius: 0px;" contenteditable="true">
  <code class=" language-html">
  <!-- My Code -->
  </code>
</pre>
```

as soon as I start typing in the editor, the `language-html` class of `<code>` is dropped. This would not be an issue if vuepress was counting on this class for its styling of vue code.

I will try to find the cause.

**Comment by koca** [comment]
Hi, glad you liked it ^^
I'll also take a look and let you know.

**Comment by elevatebart** [comment]
I found the source of the problem, the solution is a little more iffy though.
On load, `highlightAll` is used. this function uses `highlightElement` that is reponsible for adding the class. When we edit we use the pure `highlight` function which will not add the class.

2 solutions to make it homogeneous:
 - always use highlightElement and hoping it will not result in a blink effect
 - add the `language-class` manually in `utils/prism.js` safer route but a bit more hardcoded.

What do you think ?
I could cook up a PR tonight if you want.

**Comment by koca** [comment]
where is `highlightAll` ?

**Comment by elevatebart** [comment]
Right here https://github.com/PrismJS/prism/blob/3af5d744fd7359086c4a1fc5ac4b66fe4001c935/prism.js#L176

**Comment by koca** [comment]
Yes but maybe you could use `before-highlightall` hook and reset the `env.selector` to `null` so it won't highlight?

https://github.com/PrismJS/prism/blob/3af5d744fd7359086c4a1fc5ac4b66fe4001c935/prism.js#L186

**Comment by elevatebart** [comment]
That could work, but it feels a little hacky for me.

We could as well add the `language-html` on the `<pre>` tag after rendering. The `updated` hook for instance.
This would avoid the selector to pick up on it on load. 
We would trigger an update to do the actual rendering at load and at update with the same tools.

This does not save my issue though as vuepress is still counting on having both the language class on the `<pre>` and the `<code>`.

Let me make a quick PR

**Comment by elevatebart** [comment]
Done, tell me what you think ;)

**Comment by koca** [comment]
let me check :)

**Comment by koca** [comment]
just published a new version `0.3.0` Thank you :)
